backend/backup_database.sh

```diff
@@ -1,7 +1,26 @@
 #!/bin/bash
 
+# Set basic runtime settings
 echo "Environment set as: $1"
 export PATH=/home/vcap/deps/0/apt/usr/lib/postgresql/15/bin:$PATH
 date=$(date '+%Y-%m-%d-%H%M')
-python manage.py dbbackup -o "$1-db-backup-$date.psql.bin"
-python manage.py mediabackup -o "$1-media-backup-$date.tar"
+log_file="$date"_backup_log.txt
+
+# Collect metrics about what is currently in the db
+python manage.py collect_metrics >> "$log_file"
+
+# Backup the database
+echo "Database Backup File: $1-db-backup-$date.psql.bin" >> "$log_file"
+python manage.py dbbackup -o "$1-db-backup-$date.psql.bin" >&1 >> "$log_file"
+
+# Backup the media contents
+echo "Media Backup File: $1-media-backup-$date.tar" >> "$log_file"
+python manage.py mediabackup -o "$1-media-backup-$date.tar" >&1 >> "$log_file"
+
+# Upload logfile to s3://backups/logs
+python manage.py fac_s3 backups --ls --tgt "$date" >> "$log_file"
```

Comment:
This ls command will list all objects that start with the specified date. Therefore, you should include this date as the first part of the names of the backup files, rather than starting with environment and the type of backup. You can check backup_local.sh  where the naming is different.

Reply:
Ah! I see. I think we can remove this line then. If we want to ls, we can do it from cli, rather than in this backups file. The process of taking a backup doesn't really need to `ls` to see if there are backups there

Reply:
The intent is to record in the log some info about the nackup artifcat itself. Not sure why we would not do this. 
We should move the date to the front of the name.